magento/2/latest-tarball

```diff
@@ -1 +1 @@
-Magento-CE-2.4.6.tar.gz
+Magento-CE-2.4.6-p2.tar.gz
```

Comment:
you don't need this. the filenaming convention in `pubfiles` is different.

Reply:
hmm ok. So is this whole PR not needed? Or just `latest-version` file?